docs/user/config-file/v2.rst

```diff
@@ -286,6 +295,16 @@ Version specifiers for each tool. It must contain at least one tool.
 :Options: ``python``, ``nodejs``, ``rust``, ``golang``
 :Required: ``true``
 
+.. note::
+
+   Each tool has a ``latest`` option available, which refers to the latest version available on Read the Docs,
+   which may not match the latest version officially released.
+   Versions and the ``latest`` option are updated at least once every six months to keep up with the latest releases.
```

Comment:
Are we documenting what `latest` currently points to? Seems like that would be useful information, but also a pain to update.

Reply:
Was hoping that `(alias for the latest version available on Read the Docs)` on each tag would be enough given the list of all available versions.

Reply:
So it's basically the same as the first item in the list?

Reply:
Last item